### PR TITLE
Add vpc and architectures option to the creation of lambda from source

### PR DESCRIPTION
**Pull request by nnnkkk7**
**What this PR does / why we need it**:
I added vpc and architectures optional configuration to creation of lambda from source code as they did not exist.
**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**:
- **Is this breaking change**:
- **How to migrate (if breaking change)**:

### CODE REVIEW

File: pkg/app/piped/platformprovider/lambda/client.go [hunk 1]
```diff
@@ -184,11 +184,24 @@ func (c *client) CreateFunctionFromSource(ctx context.Context, fm FunctionManife
 			Variables: fm.Spec.Environments,
 		},
 	}
+	if len(fm.Spec.Architectures) != 0 {
+		var architectures []types.Architecture
```
Comment (khanhtc1202) [inline pkg/app/piped/platformprovider/lambda/client.go:188]:
```suggestion
		architectures := make([]types.Architecture, 0, len(fm.Spec.Architectures))
```

**Review by khanhtc1202 — APPROVED** [review]
Thanks 🙌

**Comment by khanhtc1202** [comment]
@nnnkkk7 the DCO check is failed, please resign to the commit 🙏

**Comment by khanhtc1202** [comment]
@ffjlabo could you check this?

**Review by ffjlabo — APPROVED** [review]
Thank you for fixing 🚀

**Pull request merged**